packages/valory/contracts/gnosis_safe/contract.py

```diff
@@ -182,7 +182,7 @@ def _get_deploy_transaction(  # pylint: disable=too-many-locals,too-many-argumen
             ledger_api.api.from_wei(account_balance, "ether"), 6
         )
         _logger.info(
-            "Network %s - Sender %s - Balance: %sΞ",
+            "Network %s - Sender %s - Balance: %s ETH",
```

Comment:
we don't have this character in any other place right?

Reply:
Nope, it was only here.